### PR TITLE
speed up to check stale metrics

### PR DESCRIPTION
**Pull request by marcolan018**
https://github.com/open-cluster-management/backlog/issues/13551
Signed-off-by: Marco <llan@redhat.com>

### CODE REVIEW

File: pkg/tests/observability_metricslist_test.go [hunk 1]
```diff
@@ -45,15 +45,15 @@ var _ = Describe("Observability:", func() {
 	It("[P2][Sev2][Observability][Integration] Should have not metrics which have been marked for deletion in names section (metricslist/g0)", func() {
 		By("Waiting for deleted metrics disappear on grafana console")
 		Eventually(func() error {
-			err, _ := utils.ContainManagedClusterMetric(testOptions, "cluster_version_payload offset 1m", []string{`"__name__":"cluster_version_payload"`})
+			err, _ := utils.ContainManagedClusterMetric(testOptions, "timestamp(cluster_version_payload) - timestamp(cluster_version_payload offset 1m) > 59", []string{})
```
Comment (clyang82) [inline pkg/tests/observability_metricslist_test.go:48]: less than 59 or great than 59? if the metrics disappears, the timestamp should be less than 59 in my understand.

Comment (marcolan018) [reply pkg/tests/observability_metricslist_test.go:48]: @clyang82 
yes, so if metrics disappear, the value less than 59, then this query expression should find nothing and return error. And the error should be caught outside by "Should(MatchError("Failed to find metric name from response")", then the test case success.

btw, the e2e test case fail and I am still investigating

**Comment by clyang82** [comment]
💯 
/lgtm

**Comment by clyang82** [comment]
@morvencao @songleo PTAL

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**